Introduce alert-message component

AshTNA opened the pull request:
Hi @gtvj 

Here is the PR for the alert message component, relating to https://national-archives.atlassian.net/browse/DS-233 

![image](https://user-images.githubusercontent.com/8880610/123625975-38d8c380-d808-11eb-95ef-c45ecb91ce1f.png)


I have checked it on:

- Chrome, MacOS X
- Firefox, Windows 10
- IE11, Windows 10

A couple questions:

- Do these need a `role="alert"`? VoiceOver seemed to read them out first anyway, since they're the first elements on the page.
- Should I ask a designer to look over this before merging?

Thanks :+1:

gtvj commented:
Hi @AshTNA - I don't think it does need the `alert` role because that's for dynamically updated content (and I believe the intention is that these will be included in the initial HTML. 

What is the IE bug with `rem`? We're using the `rem` measurement throughout Etna?

AshTNA commented:
Thanks @gtvj - I can't seem to replicate the issue on IE11 anymore. When I was loading  the  page in IE11, the `rem` padding was not being  applied, even though it existed in DevTools. It only kicked in when I unchecked it, and checked it again in DevTools. I switched  it to `em` and then the issue stopped happening. I switched it back to `rem` just now to try and surface the bug again, but it's not having any problems with `rem` anymore.

gtvj commented:
Thanks Ash - if we're happy with `rem` could you revert it? I'd really like to have a single reference point for all relative measurements.

AshTNA commented:
Thanks @gtvj - I've reverted it.

gtvj commented:
Thanks @AshTNA - merging now.
